spec/controllers/case_workers/claims_controller_spec.rb

```diff
@@ -16,6 +16,7 @@
       10.times do |n|
         Timecop.freeze(n.days.ago) do
           claim = create(:allocated_claim, case_number: "A" + "#{(n+1).to_s.rjust(8,"0")}")
+          create(:misc_fee, claim: claim, quantity: n*1, rate: n*1)
           @claims << claim
```

Comment:
lets make a note in the TODO file that the number of claims per page should be parameterised, and then we can test this without creating 11 claims.